plinth/modules/config/config.py

```diff
@@ -250,6 +251,17 @@ def set_hostname(hostname):
                                      old_hostname=old_hostname,
                                      new_hostname=hostname)
 
+    # Domain name should be ASCII. If it's unicode, convert to ASCII.
+    domainname = old_domainname
+
+    LOGGER.info('Changing domain name to - %s', domainname)
+    actions.superuser_run('domainname-change', [domainname])
+
+    domainname_change.send_robust(sender='config',
```

Comment:
In this case we are not really changing the domain (it should end up the same before and after). So I don't think we need to send the domainname_change signal.